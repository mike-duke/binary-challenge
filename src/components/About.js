import React from 'react';

export const About = () => {
  return (
    <section className='about'>
      <p>My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.</p> 
      <p>My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.</p>
      <p>Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.</p> 
      <p>I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.</p>
    </section>
  )
}