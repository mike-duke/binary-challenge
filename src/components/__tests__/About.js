import React from 'react';
import { shallow } from 'enzyme';
import { About } from '../About';

describe('About', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});