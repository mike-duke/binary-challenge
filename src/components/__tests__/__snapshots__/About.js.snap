// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.
        </p>,
        <p>
          My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.
        </p>,
        <p>
          Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.
        </p>,
        <p>
          I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.
        </p>,
      ],
      "className": "about",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.",
        },
        "ref": null,
        "rendered": "My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.",
        },
        "ref": null,
        "rendered": "My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.",
        },
        "ref": null,
        "rendered": "Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.",
        },
        "ref": null,
        "rendered": "I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.",
        "type": "p",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.
          </p>,
          <p>
            My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.
          </p>,
          <p>
            Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.
          </p>,
          <p>
            I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.
          </p>,
        ],
        "className": "about",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.",
          },
          "ref": null,
          "rendered": "My son has had severe anxiety since he was very young. When he was smaller, he once described the feeling he gets as a Torcano (a mix between a tornado and a volcano). This became our code word for him to let us know when he was having an anxiety attack.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.",
          },
          "ref": null,
          "rendered": "My boys are both much older now (15 and 21), but I’ve spent all of their lives trying to help them navigate the world as best they can through ADHD, anxiety, and depression. As they were growing up, I would have liked to have a place where I could do research on these topics with the knowledge that the articles I read were from sources that other parents trust.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.",
          },
          "ref": null,
          "rendered": "Also, I’ve added an area to view current events, as I’ve found that the goings on of the world at large can have a significant effect on children with anxiety and depression.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.",
          },
          "ref": null,
          "rendered": "I hope you find something helpful here that will help you help your own children, and most of all, know that you are not alone in your search.",
          "type": "p",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
